modules/dice/process.py

```
@@ -61,7 +61,7 @@ async def process_expression(msg: Bot.MessageSession, expr: str, dc: str):
 
 def parse_dice_expression(msg: Bot.MessageSession, dices: str):
     dice_item_list = []
-    math_func_pattern = '(' + '|'.join(re.escape(func) for func in math_funcs.keys()) + ')'  # 数学函数
+    math_func_pattern = '(' + '|'.join(re.escape(func) for func in math_funcs) + ')'  # 数学函数
```

Ruff (0.8.0)

`re` may be undefined, or defined from star imports

(F405)